cquery.el

```diff
@@ -99,6 +94,23 @@ does not belong to the current workspace.
   :type '(repeat directory)
   :group 'cquery)
 
+(defcustom cquery-project-root-matchers
+  '(projectile-project-root "compile_commands.json" ".cquery")
```

Comment:
".cquery" overrides "compile_commands.json" on the server side.

Reply:
Ah, thanks, I'll swap them around. I was preserving the order of the old `cquery--get-root`, but I'll change it so that it matches the executable behaviour.

Reply:
Thinking it over, "compile_commands.json" before ".cquery" is reasonable.

For people have hierarchical ".cquery" setting, they can put a compile_commands.json in the project root because otherwise .cquery in subdirectories will mark the :rootUri 

A bit weird though